### DISCUSSION TITLE
Preview Tool of Geoserver in GeOrchestra environment does not work

### DISCUSSION THREAD

**Discussion by haytastan** [georchestra/docker#327] [Q&A] [answered]
**Preview Tool** of Geoserver in GeOrchestra environment **does not work** for the OpenLayers and PDF formats.
As for the other formats to preview, there seems no problem.

The error message I got is as below:

**"java.lang.OutOfMemoryError: unable to create native thread: possibly out of memory or process/resource limits reached
unable to create native thread: possibly out of memory or process/resource limits reached"**

Looking forward to the solution.
Regards,

Hayati TASTAN, PhD in GIS

**Comment by edevosc2c** [comment, marked as answer]
Hello,
The error message speaks for itself. Geoserver is running out of free memory.
Make sure to have at the very least 20GB of RAM on your computer.

**Comment by haytastan** [comment]
Thanks,
I'll check the RAM of the Virtual Ubuntu Server. 
Regards,
